bin/crank

```diff
@@ -1,8 +1,6 @@
-#!/bin/bash
-# HUH? https://www.cyberciti.biz/tips/finding-bash-perl-python-portably-using-env.html
+#! /usr/bin/env bash
 
 # Thank TOM for ./crank
```

Comment:
@snuggs There *is* a way to do allow you do horrible things to master.

But you don't want to do that. 😄